src/frontend/admin/src/components/templates/courses/form/sections/product-relation/CourseProductRelationRow.tsx

```diff
@@ -205,6 +210,18 @@ export function CourseProductRelationRow({
         enableEdit={canEdit}
         enableDelete={canEdit}
         disableDeleteMessage={disabledActionsMessage}
+        permanentRightActions={
+          <MenuPopover
+            id={`course-product-relation-actions-${relation.id}`}
+            menuItems={[
+              {
+                title: intl.formatMessage(commonTranslations.copyUrl),
+                icon: <LinkIcon fontSize="small" />,
```

Comment:
Instead of LinkIcon, I'll use the CopyAll or something similar.
<img width="121" alt="image" src="https://github.com/openfun/joanie/assets/9265241/3a09cda7-1576-4677-81e5-6f04afe23ab8">